modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZCardsParagraphBehavior.php

```diff
@@ -59,6 +59,20 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       '#description' => $this->t('Select a card title style.'),
     ];
 
+    $form['card_title_level'] = [
+      '#title' => $this->t('Card title level'),
+      '#type' => 'select',
+      '#options' => [
+        'h2' => $this->t('Section heading (H2)'),
```

Comment:
In the card deck edit form, the text that can be visually seen in the dropdown before clicking the dropdown is not very helpful with the current labeling here.
![image](https://user-images.githubusercontent.com/4400438/200090782-3c981ee1-a818-4735-9171-92884cf7501f.png)

They all say "Subsection he..." so I have to click into it to see what the heading actually is. Could the heading level be moved to the front of these labels, since they're the most pertinent information in the label? I would suggest something like "H2 Card title heading" or even just "H2".

I know this PR is currently using the same verbiage used for the Text on Media's Title heading level field, and it's a usability issue there as well.